showpass/Showpass-E2E-tests
# Modify the 'Sign out' methods both in web and mobile view

**Description:**
Just noticed that a method which is used to sign out when a user is unexpectedly logged in is not doing a good job.
Needed to split a single method and make two separate methods for web view and mobile view.
Ran both. Both worked fine.

**How to test:**
- Find a test which has the cy.logIntoPortal() method
- Place an additional method before cy.logIntoPortal() to log into the portal so that the user stays logged in
- Run the test
- Ensure the script is signing out each time it sees that it's logged in unexpectedly 

**Test execution results:**
See Cypress Cloud for details [HERE](https://cloud.cypress.io/projects/e9axi7/runs/3a6c79a0-1e46-41da-b174-d4f92b64fd41/test-results/a08f1821-c8b3-4d06-a957-5d95da69494b)
